packages/web-app-files/src/components/Search/List.vue

```diff
@@ -167,6 +167,9 @@ export default defineComponent({
         itemCount: this.itemCount,
         totalResults: this.searchResult.totalResults
       })
+    },
+    routeQueryTerm() {
+      return this.$route.query.term
```

Comment:
Was there any reason for this?

Reply:
Maybe because $route was and is not globally available, let's export it from setup() instead